src/main/resources/rules.ini

```diff
@@ -0,0 +1,89 @@
+##########################
+## Dune II - The Maker
+##
+##########################
+
+
+[WEAPONS]
+
+[WEAPONS/RIFLE]
+image=projectiles/SmallBullet.png
+width=6
+height=6
```

Comment:
I thought about that, but we can only calculate the width when we know the amount of facings (ie, `width` (of one bullet) could be `width/facings`). And `height` could be picture height or `height/animations` or something.